server.lua

```diff
@@ -17,6 +17,15 @@ QBCore.Commands.Add("dev", "Enable/Disable developer Mode", {}, false, function(
     TriggerClientEvent("qb-admin:client:ToggleDevmode", source)
 end, 'admin')
 
+local notified = false
+function notification()
+    if not notified then
+        notified = true
+        Wait(30000)
+        notified = false
+    end
+end
+
```

Comment:
Thinking out loud here - unless you are tracking the source in the notified variable, won't `notified` be set anytime _anyone_ on the server becomes stressed?